Golem.Tools/GolemRunnable.cs

```diff
@@ -28,34 +29,41 @@ protected GolemRunnable(string dir, ILogger logger)
 
         public abstract bool Start();
 
-        protected bool StartProcess(string file_name, string working_dir, string args, Dictionary<string, string> env, bool openConsole = true)
+        protected bool StartProcess(string file_name, string working_dir, string args, Dictionary<string, string> env, bool console_output = false)
         {
-            Command process = RunCommand(file_name, working_dir, args, env);
-            AddShutdownHook(process);
```

Comment:
I need it to shutdown processes when tests are failing, but I checked that yagna gets shutdown on test failure, so it is fine.